tools/control_board/requirements.txt

```diff
@@ -0,0 +1,2 @@
+github3.py
+progressbar==2.2
```

Comment:
In order to move forward with som sort of standarization, can you move this file to the root f the repository pleas, is order to allow it be used with the installer (some work is being done around that)